randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -258,7 +264,7 @@ Extra - room_id: [6, 24]
   * Extra - door_idx: (15, 48)
   > Pickup (Missile Tank)
       Any of the following:
-          Can Use Bombs
+          Can Break Single Bomb Blocks
```

Comment:
this assumes screw is enough, you would need an `and morphball` encapsulating this statement

Reply:
Updated.